Content.Server/Connection/ConnectionManager.cs

```diff
@@ -109,12 +109,19 @@ private async Task NetMgrOnConnecting(NetConnectingArgs e)
             if (_cfg.GetCVar(CCVars.PanicBunkerEnabled))
             {
                 var showReason = _cfg.GetCVar(CCVars.PanicBunkerShowReason);
+                var customReason = _cfg.GetCVar(CCVars.PanicBunkerCustomReason);
 
                 var minMinutesAge = _cfg.GetCVar(CCVars.PanicBunkerMinAccountAge);
                 var record = await _dbManager.GetPlayerRecordByUserId(userId);
                 var validAccountAge = record != null &&
                                         record.FirstSeenTime.CompareTo(DateTimeOffset.Now - TimeSpan.FromMinutes(minMinutesAge)) <= 0;
 
+                // Use the custom reason if it exists & they don't have the minimum account age
+                if (customReason != string.Empty && !validAccountAge)
+                {
+                    return (ConnectionDenyReason.Panic, customReason, null);
```

Comment:
won't this reason show even if the "showReason" value is false? It seems like you'd want to move this into the if statement below.